weblate/trans/views/basic.py

```diff
@@ -315,6 +315,12 @@ def show_translation(request, project, component, lang):
         if test_component.can_add_new_language(user):
             other_translations.append(GhostTranslation(test_component, obj.language))
 
+    # Limit the number of other components displayed to 10, preferring untranslated ones
+    other_translations = sorted(
+        other_translations,
```

Comment:
I'm not sure what `black` would change here... it seems fine to me. Any ideas why this happens?